x-pack/plugins/apm/common/environment_filter_values.ts

```diff
@@ -9,11 +9,29 @@ import { i18n } from '@kbn/i18n';
 export const ENVIRONMENT_ALL = 'ENVIRONMENT_ALL';
 export const ENVIRONMENT_NOT_DEFINED = 'ENVIRONMENT_NOT_DEFINED';
```

Comment:
Should these be renamed for clarity?
```suggestion
export const ENVIRONMENT_ALL_VALUE = 'ENVIRONMENT_ALL';
export const ENVIRONMENT_NOT_DEFINED_VALUE = 'ENVIRONMENT_NOT_DEFINED';
```

Reply:
thought: Perhaps the values and labels shouldn't be exported. Instead only the `ALL_OPTION` and `NOT_DEFINED_OPTION` are exported and then we do `ALL_OPTION.value` and `ALL_OPTION.label`?

Reply:
(not sure it actually makes a difference to readability though - just a thought)